custom_components/ge_home/update_coordinator.py

```diff
@@ -270,31 +293,35 @@ def all_appliances_updated(self) -> bool:
 
     async def on_appliance_list(self, _):
         """When we get an appliance list, mark it and maybe trigger all ready."""
-        _LOGGER.debug('Got roster update')
+        _LOGGER.debug("Got roster update")
         self.last_update_success = True
         if not self._got_roster:
             self._got_roster = True
-            #TODO: Probably should have a better way of confirming we're good to go...
-            await asyncio.sleep(5)  # After the initial roster update, wait a bit and hit go
+            # TODO: Probably should have a better way of confirming we're good to go...
+            await asyncio.sleep(
```

Comment:
I'm seeing things like this throughout - guessing you're using a code formatter.  Can you update it to not perform this particular type of reformat?